Source/Core/Common/Logging/LogManager.cpp

```diff
@@ -228,14 +228,22 @@ void LogManager::LogWithFullPath(LogLevel level, LogType type, const char* file,
   }
 }
 
-LogLevel LogManager::GetLogLevel() const
+LogLevel LogManager::GetEffectiveLogLevel() const
+{
+  return m_effective_level;
+}
+
+void LogManager::SetConfigLogLevel(const LogLevel level)
 {
-  return m_level;
+  const auto clamped_level = std::clamp(level, LogLevel::LNOTICE, MAX_EFFECTIVE_LOGLEVEL);
```

Comment:
Is this clamp actually useful? I don't see any callers that would be able to call this with an out-of-range value, and anyway it's the case that code that needs to read a clamped level needs to call GetEffectiveLogLevel instead of directly using LOGGER_VERBOSITY.